src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -106,6 +106,13 @@ class EventDetailsContent extends AsyncComponent<Props, State> {
     return this.props.eventSlug.split(':')[0];
   }
 
+  generateTagKey = (tag: EventTag) => {
+    // Some tags may be normalized from context, but not all of them are.
+    // This supports a user making a custom tag with the same name as one
+    // that comes from context as all of these are also tags.
+    return `tags[${tag.key}]`;
```

Comment:
Could we quote only when the tag overlaps a known field? The field list in `utils/discover/fields` could be used.

Reply:
like only quote on those? and not any other custom tag? that's a good idea 👍

Reply:
Yes. The quote syntax is only required when tags overlap with core event properties. Doing it all the time clutters the search bar.

Reply:
Fixed